packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -170,7 +161,7 @@ class WalletsController {
 
   public static verifyPasswordComplexity = (password: string) => {
     if (password.length < 8) {
-      return false
+      throw Error(i18n.t('messages.wallet-password-at-least-8-characters'))
```

Comment:
Please change this to:

1. Use constant for the length (`8`).
2. Use that constant from the `i18n` message.